addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -0,0 +1,592 @@
+# -*- coding: utf-8 -*-
+
+from odoo import _, models
+from odoo.tools import float_repr
+from odoo.tests.common import Form
+
+from zeep import Client
+from lxml import etree
+
+UOM_TO_UNECE_CODE = {
+    'product_uom_unit': 'C62',
+    'product_uom_dozen': 'DZN',
+    'product_uom_kgm': 'KGM',
+    'product_uom_gram': 'GRM',
+    'product_uom_day': 'DAY',
+    'product_uom_hour': 'HUR',
+    'product_uom_ton': 'TNE',
+    'product_uom_meter': 'MTR',
+    'product_uom_km': 'KTM',
+    'product_uom_cm': 'CMT',
+    'product_uom_litre': 'LTR',
+    'product_uom_cubic_meter': 'MTQ',
+    'product_uom_lb': 'LBR',
+    'product_uom_oz': 'ONZ',
+    'product_uom_inch': 'INH',
+    'product_uom_foot': 'FOT',
+    'product_uom_mile': 'SMI',
+    'product_uom_floz': 'OZA',
+    'product_uom_qt': 'QT',
+    'product_uom_gal': 'GLL',
+    'product_uom_cubic_inch': 'INQ',
+    'product_uom_cubic_foot': 'FTQ',
+}
+
+COUNTRY_EAS = {
```

Comment:
reuse the one in bis3?

Reply:
Yet another dependency ? (and the answer depends also on the comment below)